met-api/src/met_api/models/survey.py

```diff
@@ -92,6 +92,7 @@ def update_survey(cls, survey: SurveySchema) -> DefaultMethodResult:
             form_json=survey.get('form_json', None),
             updated_date=datetime.now(),
             updated_by=survey.get('updated_by', None),
+            name=survey.get('name', ''),
```

Comment:
name=survey.get('name', ''),

incase if UI fails to send , it might become nulll..
to avoid it , https://github.com/bcgov/met-public/pull/511 see this

Reply:
added the record check to prevent the null case